src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1666,28 +1662,35 @@ inline RppStatus rpp_store4_f32pln3_to_f32pkd3(Rpp32f* dstPtr, __m128* p)
     return RPP_SUCCESS;
 }
 
-inline RppStatus rpp_bilinear_load4_f16pkd3_to_f32pln3(Rpp16f* srcPtrTopRow, Rpp16f* srcPtrBottomRow, Rpp32u* loc, __m128* p)
+inline RppStatus rpp_store4_f32pln1_to_f32pln1(Rpp32f* dstPtr, __m128 p)
```

Comment:
This function is already available as rpp_store4_f32_to_f32(). Please call the same. Similar comment for the two functions above this.
Your rpp_store4_f32pln3_to_f32pln3() is already available as rpp_store12_f32pln3_to_f32pln3().
Your rpp_store4_f32pln3_to_f32pkd3() is already available as rpp_store12_f32pln3_to_f32pkd3().